workflows/sock-shop-demo/usingCmdProbe/workflow_cron.yaml

```diff
@@ -52,379 +46,13 @@ spec:
-                        resources:
-                          - "jobs"
-                          - "pods"
-                          - "pods/log"
-                          - "replicationcontrollers"
-                          - "deployments"
-                          - "statefulsets"
-                          - "daemonsets"
-                          - "replicasets"
-                          - "deploymentconfigs"
-                          - "rollouts"
-                          - "pods/exec"
-                          - "events"
-                          - "chaosengines"
-                          - "chaosexperiments"
-                          - "chaosresults"
-                        verbs:
-                          - "create"
-                          - "list"
-                          - "get"
-                          - "patch"
-                          - "update"
-                          - "delete"
-                          - "deletecollection"
-                    image: "litmuschaos/go-runner:latest"
-                    imagePullPolicy: Always
-                    args:
-                    - -c
-                    - ./experiments -name pod-network-loss
-                    command:
-                    - /bin/bash
-                    
-                    env:
-                    - name: TARGET_CONTAINER
-                      value: ''
-                    - name: LIB_IMAGE
-                      value: 'litmuschaos/go-runner:latest' 
-                    - name: NETWORK_INTERFACE
-                      value: 'eth0'
-                    - name: TC_IMAGE
-                      value: 'gaiadocker/iproute2'
-                    - name: NETWORK_PACKET_LOSS_PERCENTAGE
-                      value: '100' #in PERCENTAGE
-                    - name: TOTAL_CHAOS_DURATION
-                      value: '30' # in seconds
-                    - name: RAMP_TIME
-                      value: ''
-                    - name: LIB
-                      value: 'litmus'
-                    - name: PODS_AFFECTED_PERC
-                      value: ''
-                    - name: TARGET_PODS
-                      value: ''
-                    - name: CONTAINER_RUNTIME
-                      value: 'docker'
-                    - name: DESTINATION_IPS
-                      value: ''
-                    - name: DESTINATION_HOSTS
-                      value: ''
-                    - name: SOCKET_PATH
-                      value: '/var/run/docker.sock'
-                    - name: SEQUENCE
-                      value: 'parallel'
-                    labels:
-                      name: pod-network-loss
-      container:
-        image: lachlanevenson/k8s-kubectl
-        command: [sh, -c]
-        args: [ "kubectl apply -f /tmp/pod-network-loss.yaml -n {{workflow.parameters.adminModeNamespace}}",]
+          - "kubectl apply -f https://hub.litmuschaos.io/api/chaos/1.13.0?file=charts/generic/experiments.yaml -n
```

Comment:
Can we use master version here? we can use {branch,version} mapping to avoid the vendor mismatch problem

Reply:
yes updated it